Simplified/Accounts.swift

```diff
@@ -191,7 +191,7 @@ final class Account:NSObject
     }
   }
   
-  func getLicenseURL(_ type: URLType) -> URL? {
+  @objc func getLicenseURL(_ type: URLType) -> URL? {
```

Comment:
If I'm reading the proposal correctly, it seems like `@objcMembers` is a new keyword assigned to what `@obj` _used_ to do at the class level. Does this function work without the `@objc` marker? It's using an enum that has also been marked `@objc`
@winniequinn